kik_unofficial/datatypes/xmpp/chatting.py

```diff
@@ -60,43 +60,56 @@ class OutgoingChatImage(XMPPElement):
-            '<disallow-save>false</disallow-save>'
-            '<file-content-type>image/jpeg</file-content-type>'
+            f'<message to="{self.peer_jid}" id="{self.message_id}" cts="{timestamp}" type="{message_type}" xmlns="{xmlns}">'
+            f'<pb/>'
+            f'<kik push="true" qos="true" timestamp="{timestamp}" />'
+            f'<request xmlns="kik:message:receipt" d="true" r="true" />'
+            f'<content id="{self.content_id}" app-id="{app_id}" v="2">'
+            f'<strings>'
+            f'<app-name>{app_name}</app-name>'
+            f'<file-size>{self.parsed_image["size"]}</file-size>'
+            f'{image_config}'
+            f'<file-content-type>image/jpeg</file-content-type>'
             f'<file-name>{self.content_id}.jpg</file-name>'
-            '</strings>'
-            '<extras />'
-            '<hashes>'
-            f'<sha1-original>{self.parsed["SHA1"]}</sha1-original>'
-            f'<sha1-scaled>{self.parsed["SHA1Scaled"]}</sha1-scaled>'
-            f'<blockhash-scaled>{self.parsed["blockhash"]}</blockhash-scaled>'
-            '</hashes>'
-            '<images>'
-            f'<preview>{self.parsed["base64"]}</preview>'
-            '<icon></icon>'
-            '</images>'
-            '<uris />'
-            '</content>'
-            '</message>'
+            f'</strings>'
+            f'<extras />'
+            f'<hashes>'
+            f'<sha1-original>{self.parsed_image["SHA1"]}</sha1-original>'
+            f'<sha1-scaled>{self.parsed_image["SHA1Scaled"]}</sha1-scaled>'
+            f'<blockhash-scaled>{self.parsed_image["blockhash"]}</blockhash-scaled>'
+            f'</hashes>'
+            f'<images>'
+            f'<preview>{self.parsed_image["base64"]}</preview>'
+            f'<icon>iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAAXNSR0IArs4c6QAAAARzQklUCAgICHwIZIgAAAMpSURBVGiB7ZdNSBRhGMd/M1NJhZaGlRHmIRULsmN0sGsXqUOnoEMQhF07WRB0SPuAkjBCSTzUIfDS0sEgsCw6lVQmmbrq+oHa5k7q6urqzkyHSXTdHZuvnF3Z32n2fR/m/f/3fd73eUZA04SydxxWFao1OAPkkdrIAvhEidruCvxCWbtWrCi8BEq9VmaRHkmiUlQVqkk/8QClqkK1UNKmhUj9tDFCFklf8QB5otcKnJIx4DUZA16zxWjidD5cL4a9WRspJ5FgFGr6oPVX8nnDHUgF8aBruFZsPG9oIBXEL7OelrQ/AxkDXuO6gZgG84rbbzXG8Bq1w6cpeDgIEQVO5MKVItghublCIq4ZGIrA1e8wEdV/d4Vh91a4VBgfp2owp0C2Syu7lkKd4RXxABrwdDQxbmQeqr7B2II767pmQEg2tmZQ0aBhGD5OwYMBWFKdr+uagWM5ULCq4AjAhYPxMa+C8GJCf34bgt455+s6NjC+AFEVCrdD3VE4mQvlOVBVBOcPrMT9ikJ9QN8FgJkYPArot5YTHB2l0QW4+AUuH4JzBXB8Fzwp10Vlrflrno9DIBI/9mYSWoNQuc++Bts7EFXhrh+G5/Vuse9vOkhCoviuMDQEYG3Kq0DjEPxesqvCgYGvM/Be1p/nFLjZC9NJhMzE4LYflgxSpT8Crw1aZTPYMjCvwL1+vWAt0zEFvp+JsW2T8Hna+F2KBnWDEFq0o8SGAQ1oGoHOmfhxFagbgI5VYscX4I7/3wc1tAj3B1YOuBUsGwhEoGUs+dycArf69JyOqnrqyCbzu20SematqrFhoGk4vuKu5UcYno3qKdUum3+vvASNw1bVWLhGNeCDDC3j68cpwOMhvahZ7Upbg3B2P5zak7yyJ8P0DszGoH7QXKyi6TXCDo1DEImZjze9AzsluFECIQd3thnyt1lrwU0bEAU4km1H0v8l80npNYYGgutclRvNeloMDdT0pYaJYFQvjkYIJW2aw47cWzbvGUgXMga8ZlMYsND0phyyKIDPaxV2EcAnihK1QI/XYmzQI0rUit0V+CWJSgGaSY90kgVoliQquyvw/wEBywt7TQ67XwAAAABJRU5ErkJggg==</icon>'
```

Comment:
What's this hardcoded `icon`?

Reply:
I don't remember why that is there exactly. I do know if it is not there the image does not display properly. You can not click on the image to zoom in.

Reply:
What image does this icon decode to?

Reply:
Here is what it decodes to. 
![image](https://github.com/tomer8007/kik-bot-api-unofficial/assets/27271698/8e356b87-55c2-4f95-bae7-1f3f7e7426f7)